File: SocialNetworks/app/src/main/java/com/flatstack/socialnetworks/ui/MainActivity.java
package com.flatstack.socialnetworks.ui;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.flatstack.socialnetworks.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportFragmentManager()
            .beginTransaction()
            .addToBackStack(null)
            .replace(R.id.container, new MainScreen())
            .commit();
    }

}

File: kotlin-base/app/src/main/java/com/flatstack/android/main_screen/MainActivity.kt
package com.flatstack.android.main_screen

import android.os.Bundle

import com.flatstack.android.utils.ui.BaseActivity
import com.flatstack.android.R
import com.flatstack.android.utils.ui.UiInfo

class MainActivity : BaseActivity() {

    override val uiInfo = UiInfo(layoutRes = R.layout.activity_main, titleRes = R.string.app_name,
            hasBackButton = true)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.content, MainFragment())
                    .commit()
        }
    }
}